lib/adhearsion/events.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'has_guarded_handlers'
 require 'singleton'
-require 'celluloid'
+require 'concurrent/executor/thread_pool_executor'
```

Comment:
@kares Thank you so much for this.  Can you add an entry in the CHANGELOG.md for the changing of the Events subsystem from Celluloid -> concurrent-ruby?